src/test/resources/application-integration.yml

```diff
@@ -17,6 +17,12 @@ janitor:
     resource-credential-file-path: rendered/cloud-access-sa-account.json
     resource-project-id: terra-janitor-test
     track-resource-topic-id: crljanitor-toolsalpha-pubsub-topic
-    azure-tenant-id: 0cb7a640-45a2-4ed6-be9f-63519f86e04b
-    azure-subscription-id: 3efc5bdf-be0e-44e7-b1d7-c08931e3c16c
-    azure-managed-resource-group-name: mrg-terra-integration-test-20211118
+    # Reusing static MRG from https://github.com/broadinstitute/terra-github-workflows/blob/main/.github/workflows/attach-billing-project-to-landing-zone.yaml
```

Comment:
fyi (and you probably know already). Those are configs for janitor integration test. 
The "production Janitor" config is in vault: https://github.com/broadinstitute/terra-helmfile/blob/04733580a5fbde7141a4f0378c9763525009c1d1/charts/crljanitor/values.yaml#L113

Reply:
Yup! I just intended here to update the integration test config (the old MRG was created in 2021).